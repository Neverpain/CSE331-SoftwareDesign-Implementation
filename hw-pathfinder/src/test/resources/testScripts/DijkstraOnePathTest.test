# Tests that Dijkstra's algorithm works on two nodes with
# only one edge between them, outputs it

CreateGraph g

AddNode g A
AddNode g B

AddEdge g A B 1.0

FindPath g A B