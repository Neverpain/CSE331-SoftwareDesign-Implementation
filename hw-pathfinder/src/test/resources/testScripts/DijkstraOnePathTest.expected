# Tests that Dijkstra's algorithm works on two nodes with
# only one edge between them, outputs it

created graph g

added node A to g
added node B to g

added edge 1.000 from A to B in g

path from A to B:
A to B with weight 1.000
total cost: 1.000